src/app/submission/import-external/submission-import-external.component.ts

```diff
@@ -64,7 +66,7 @@ export class SubmissionImportExternalComponent implements OnInit, OnDestroy {
    * The initial pagination options
    */
   public initialPagination = Object.assign(new PaginationComponentOptions(), {
-    id: 'submission-external-source-relation-list',
+    id: 'spc',
```

Comment:
Unfortunately, this components partially uses the SearchConfiguration to retrieve its pagination information. Therefore in this case the id needs to be the same as the one in the SearchConfiguration. However, this does not interfere with any other components that might have the same id since they are displayed on different pages.